first_file.py

```diff
@@ -0,0 +1,57 @@
+field_array = []
+
+#perilambanei kai _ kai arithmous
+#function that checks if input_str consists of latin chars
```

Comment:
Move this comments in [docstrings](https://www.python.org/dev/peps/pep-0257/#what-is-a-docstring) please. Docstrings are used in Python to describe what a function, class or module does.

It should look like this:

```python
def check_latin_chars(input_str):
    """
    perilambanei kai _ kai arithmous
    function that checks if input_str consists of latin chars
    """
```